setup.md

```diff
@@ -0,0 +1,334 @@
+
+If you want to delete (remove) a file do:
+
+```
+rm file
+```
+
+If you want to rename a file do:
+
+```
+mv file newfilename
+```
+
+If you want to print the contents of a file do:
+
+```
+more file
+```
+
+
+### Basic Windows Anaconda terminal instructions
+
+This section only provides a very basic overview of using the Anaconda terminal.
+For more information please see the following tutorials:
+
+- [Windows command line tutorial](https://youtu.be/MBBWVgE0ewk) (youtube video)
+  Note: this uses the standard windows command line, but the instructions should carry over
+  to the Anaconda terminal.
+  Disclaimer: we have not watched all 9 videos in the tutorial series and do not take
+  responsibility for any of the contents.
+
+**_On first opening the terminal_**
+
+Generally upon opening the terminal you will be placed in your %HOMEPATH% directory.
+This is the base directory where all the main folders (My Documents, Downloads, etc...)
+for your login account are placed, which is usually `C:\Users\username`. If you ever
+need to return to this folder enter the following command:
+
+```
+cd %HOMEPATH%
+```
+
+**_Seeing and navigating the contents of folders_**
+
+One of the main uses of the terminal is to allow users to quickly see and navigate
+through the various files and folders on your computer.
+
+You can see what files are in the current folder you are accessing using:
+
+```
+dir
+```
+
+You can change from one folder to another (in this case named 'newfolder') by doing:
+
+```
+cd newfolder
+```
+
+If you want to know where you are amongst all your folders you can do:
+
+```
+cd
+```
+
+This will give you the full path to the current folder.
+
+If you want to delete (remove) a file do:
+
+```
+del file
+```
+
+If you want to rename a file do:
+
+```
+rename file newfilename
+```
+
+If you want to print the contents of a file do:
+
+```
+more file
+```
+
+### Using editors
+
+At several points during the tutorials you might be asked to edit a file.
+
+There are several editors available and generally we recommend you use
+the one that you are most accustomed with (if possible).
+
+Here are our recommendations for editors that can be invoked directly from
+the command line:
+
+#### linux/macOS
+
+- [gedit](https://help.gnome.org/users/gedit/stable/) (recommended for linux, usually pre-installed)
+- [nano](https://wiki.gentoo.org/wiki/Nano/Basics_Guide) (recommended for linux & macOS, pre-installed)
+- [TextEdit](https://support.apple.com/en-gb/guide/textedit/welcome/mac) (for macOS, pre-installed.  From the terminal prompt you can issue `open -a TextEdit`)
+- [vim](https://www.vim.org/download.php)
+
+#### Windows
+
+- notepad (recommended)
+
+Built into Windows, just type `notepad filename` in your terminal. This will open a
+window where you can edit the file, save and then close the window to get back to
+your terminal.
+
+- [notepad++](https://notepad-plus-plus.org/)
+- [vim](https://www.vim.org/download.php)
```

Comment:
Should we say to use their preferred editor and link to a cross-platform one otherwise? Giving this much choice could end up with a lot of "how I do X in editor Y" type of questions.

Reply:
See line 202 (can change if you think it needs more details)

Reply:
That's fine. But if people won't use their editor of choice, I think we should point to a single multi-platform one.

Reply:
You mean... vim 😱 ?

Imho the simplest case is if we can do gedit/textedit/notepad, unless you have something else in mind (Windows is the pain here, I don't think there's an easy way to get nano installed, and honestly more "install software X" instructions is probably going to cause us trouble).